CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 *[CalVer, YY.month.patch](https://calver.org/)*
 
+## 22.7.5
```

Comment:
[Note to self: do not merge this before releasing .4 or we'll end up with out-of-order releases 😱]